scripts/runfabtests.sh

```diff
@@ -284,11 +290,60 @@ function cs_test {
 	fi
 }
 
+function complex_test {
+	local test=$1
+	local config=$2
+	local test_exe="fi_${test}"
+	local start_time
+	local end_time
+	local test_time
+
+	# TODO: Add a better way to get to $datarootdir
+	if [ ! -f "${BIN_PATH}../share/fabtests/test_configs/${PROV}/${config}.json" ]; then
```

Comment:
Yes, ubertest knows where to find the config file and it gracefully fails if it can't find the file. But that'll mark the test as "Fail". I wanted to mark it "Not Run" from runfabtests.sh when it a config file doesn't exist for a particular provider. If it's not a big issue, I can remove the check.